src/clientsettingsdlg.cpp

```diff
@@ -228,11 +228,17 @@ CClientSettingsDlg::CClientSettingsDlg ( CClient*         pNCliP,
     butDriverSetup->setToolTip ( strSndCardDriverSetupTT );
 
     // fancy skin
-    cbxSkin->setWhatsThis ( "<b>" + tr ( "Skin" ) + ":</b> " + tr (
+    lblSkin->setWhatsThis ( "<b>" + tr ( "Skin" ) + ":</b> " + tr (
         "Select the skin to be used for the main window." ) );
 
     cbxSkin->setAccessibleName ( tr ( "Skin combo box" ) );
 
+    // Interface Language
```

Comment:
If this is new, it's going to need calling out to the translators, as the cut's been taken.

Reply:
> If this is new, it's going to need calling out to the translators, as the cut's been taken.

As far as I can see, this string does not appear in 3.8.0beta1 (even in other places), so it is new, yes.

I'd rather not restart translations for that. One option would be to merge this explicitly after all other translation PRs have been merged. This has the obvious downside that the new strings will not be translated for 3.8.0, but as it is only about a rather hidden text and hasn't been working before either, I think it would be acceptable to do so.

I'm therefore not approving yet.

Reply:
Yes, the Language whatsthis is new.
Other possibility is I take it out now, as it didn't exist before, and try to remember to put it back in for the next version.

Reply:
I agree with @hoffie - leave this in, and merge it after the translations. I also have a PR #1629 for the `.ts` files and can deal with them both at the same time.